src/components/login/LoginForm.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from "react";
+import { Button, FormGroup, FormControl, ControlLabel } from "react-bootstrap";
+import "./LoginForm.css";
+
+export default class LoginForm extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            email: "",
+            password: ""
+        };
+    }
+
+    validateForm() {
+        return this.state.email.length > 0 && this.state.password.length > 0;
+    }
+
+    handleChange = event => {
+        this.setState({
+            [event.target.id]: event.target.value
```

Comment:
`[event.target.id]` **this is very very very bad** practice, please forget that)) 
If you want single handler for different fields please use currying.
Here example what you need: [example](https://medium.freecodecamp.org/the-best-way-to-bind-event-handlers-in-react-282db2cf1530) last part: **Dynamic binding with the Arrow Function for multiple elements**

Reply:
i removed this logic and in next version i will add implementation based on your advice